src/vivarium_gates_mncnh/components/delivery_facility.py

```diff
@@ -0,0 +1,50 @@
+from vivarium import Component
+from vivarium.framework.engine import Builder
+from vivarium.framework.event import Event
+
+from vivarium_gates_mncnh.constants.data_values import (
+    COLUMNS,
+    DELIVERY_FACILITY_TYPE_PROBABILITIES,
+    DELIVERY_FACILITY_TYPES,
+    PREGNANCY_OUTCOMES,
+    SIMULATION_EVENT_NAMES,
+)
+from vivarium_gates_mncnh.utilities import get_location
+
+
+class DeliveryFacility(Component):
+    """ "Component that stores functionality for the delivery facility choice model."""
+
+    @property
+    def columns_required(self) -> list[str]:
+        return [COLUMNS.DELIVERY_FACILITY_TYPE, COLUMNS.PREGNANCY_OUTCOME]
```

Comment:
What component initializes the delivery facility type column, and why isn't it this one?